pkg/apis/operator/v1alpha1/tektonconfig_validation.go

```diff
@@ -43,6 +47,32 @@ func (tc *TektonConfig) Validate(ctx context.Context) (errs *apis.FieldError) {
 		}
 	}
 
+	// validate SCC config
+	if IsOpenShiftPlatform() && tc.Spec.Platforms.OpenShift.SCC != nil {
+		defaultSCC := PipelinesSCC
+		if tc.Spec.Platforms.OpenShift.SCC.Default != "" {
```

Comment:
SCC is pointer we need to check before accessing isn't it :thinking: or is it working for you ? or any defaulting done for SCC :thinking: